Example/PaymentSheet Example/PaymentSheetUITest/PaymentSheetUITest.swift

```diff
@@ -1638,7 +1638,7 @@ class PaymentSheetCVCRecollectionUITests: PaymentSheetUITestCase {
         payButton.tap()
 
         let successText = app.staticTexts["Success!"]
-        XCTAssertTrue(successText.waitForExistence(timeout: 10.0))
```

Comment:
10 seconds already feels like a long time, and hesitant to bump up to 60s.

Did we increase the time because we thought this test was "flakey?"  If so, my guess of why it is flake is due to not using `waitForExistanceAndTap()` on `app.buttons["Pay $50.99"]`

Reply:
It was for a flaky test, but I think I actually changed the timeout length in a place I didn't mean to, so I'll revert that. Thanks for catching it!

Reply:
The actual thing that it was flaky on was [this line](https://github.com/stripe/stripe-ios/blob/master/Example/PaymentSheet%20Example/PaymentSheetUITest/XCUITest%2BUtilities.swift#L311)

Reply:
Do you think it's worth extending the timeout on that one?

Reply:
No, I dont think we should update that-- Do you remember which test was flakey?  Maybe there is something we were doing before calling reload which made reload flakey?

Reply:
It was testCVCRecollectionComplete_deferredCSC()